core/src/test/java/io/questdb/cairo/AbstractCairoTest.java

```diff
@@ -250,6 +269,8 @@ public long getWriterCommandQueueSlotSize() {
     public static void tearDownStatic() {
         snapshotAgent = Misc.free(snapshotAgent);
         engine = Misc.free(engine);
+        backupDir = null;
```

Comment:
I'm not sure this is the best place to cleanup these.

Reply:
Looks ok to me .